### PR TITLE
Fix for unopened roll-call bug

### LINKED ISSUE #1114: [BUG] PoP token is non retrievable when there is an unopened roll-call
### Description (Actual behavior)
When the organizer creates a roll-call but does not open it, the pop token from subsequent roll-call is not retrieved and therefore prevents the user from using any feature that requires a pop token. That is most of them

### Expected behavior
We expect the user to be able to use the pop token from the lastly closed roll-call

### How to reproduce
1. Create a roll-call
2. Create open and close another rc afterwards
3. Create an election
4. See that user is unable to vote

### Version & Environment
This bug was reproduced on:
- Commit: [e6f7dc7](https://github.com/dedis/popstellar/commit/e6f7dc75ec530dc8fcb56ee8be52838011c7564f)

##### Front-ends:
- [ ] Fe1-Web (please include browser's names & version)
- [x] Fe2-Android (please specify if phone or emulation, and Android version)
- [ ] Not applicable

##### Back-ends:
- [ ] Be1-Go
- [ ] Be2-Scala
- [x] Not applicable

##### Environment (as applicable):
 - Device: [e.g. Pixel 4 emulator on Windows 10]

### Workaround
Always open and close created roll-call before creating another

### Possible root cause
My best guess is that the comparator logic used to determine which is the last roll-call is faulty and return the unopened roll-call as the last.

### PR DESCRIPTION
**Pull request by Warhedgehog**
this fixes #1114

### CODE REVIEW

**Review by zoemarin — APPROVED** [review]
👍

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/security/KeyManager.java [hunk 1]
```diff
@@ -113,6 +115,7 @@ public PoPToken getValidPoPToken(Lao lao) throws KeyException {
     // Find the latest closed RollCall and use the wallet to retrieve the key
     RollCall rollCall =
         lao.getRollCalls().values().stream()
+            .filter(RollCall::isClosed)
```
Comment (GabrielFleischer) [inline fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/security/KeyManager.java:118]: Yes ! Nice catch !

**Review by GabrielFleischer — APPROVED** [review]
Great !

Nice catch

**Review by ahominal — APPROVED** [review]
Seems good !

**Pull request merged**